### ISSUE TITLE
Name the App

### ISSUE THREAD

**Issue by rustyeddy** [mobilerobot-io/clowdops.net#11]
Name the App (Tool).  This one is harder to me as there is more ground to cover

o   “Mobile (Agile) Robot No-touch (or Dynamic) Cloud App Health Maintenance Process”

**Comment by rustyeddy** [comment]
Yes, I agree with both, naming the app and the platform.  But cant come up with a good descriptive name.

Though I was thinking something like "transparent hosting" or similar.  Meaning a couple things, we are transparent with our processes, and hence the client has visibility into how we work and how the project is moving.

Second, all scripts we develop are not proprietary or secret, they will always have access and be able to use them, even after they stop paying.  

just a thought.

**Comment by jteeling** [comment]
Good morning Rusty,
I think going skinny or naked as you are suggesting has a lot of merits as well for a lot of reasons.  To me the most important factor here isn't a fancy name, but, tying the name of the infrastructure and the app to the known name for brand awareness/confidence in the service.  To that end, how about:

"Mobile Robot Transparent Platform (MRTP)"  which kind of makes it self descriptive to people who kinda get it.

"Mobile Robot Transparent Monitor (MRTM)"  Who it is and what it does in simple terms.

**Comment by rustyeddy** [comment]
Totally agree with the need for a name, and something that I could way overthink it ..   But I wont.  I'll throw out one more thought (based on some underlying philosophy).

These days, it is progressively difficult (as it should be) to separate the application from it's supporting infrastructure, in particular when you consider deployment.   Likewise, it becomes much easier (and more efficient) to define every infrastructure from the view of the application or service.

Rather than view the infrastructure as a group of resources that support a group of applications and services, we will toward a 1-1 definition of "application or service" with infrastructure.  

Now reality always messes up idealistic plans, but I think this is a very important paradigm.  It also leads to the notion that:

  app + infrastructure + deployment = entire lifecycle

So I guess you can say we support the "application lifecycle",  errr, the "transparent application lifecycle" . :)

> except maybe, for shrink wrap software  

So, there are my last thoughts. ..

At anyrate, I agree!  Name it sooner rather than later.  Besides (it can always be tweeked if it needs to be).

Alright.  You get this one.   I'm assigning it to you!  :)

**Comment by jteeling** [comment]
Very interesting thoughts and I'm there with you on the concept/methodology.
What I think you actually just did was name the SPA = "Mobile Robot Transparent Application"
In my mind it probably has three components that live on the one-page diagram under the "Mobile Robot Transparent Application", those are paraphrasing:
"The Platform""The Monitor""The Deployment/Update/Repair" that completes the feedback loop or life-cycle.
I think initially it needs to be kept traditional in name even though the underlying platform and application concepts are fairly rad.  At the end of the day, customers need to be able to recognize and understand what is is and does.  At some point marketing catches up with the service and it can evolve at that point into something more cutting edge/futuristic.
That's the way I see it today.  Will update the Github shortly.  Hope you weekend is good,
John


    On Saturday, December 1, 2018, 8:42:16 AM PST, Rusty Eddy <notifications@github.com> wrote:  
 
 
Totally agree with the need for a name, and something that I could way overthink it .. But I wont. I'll throw out one more thought (based on some underlying philosophy).

These days, it is progressively difficult (as it should be) to separate the application from it's supporting infrastructure, in particular when you consider deployment. Likewise, it becomes much easier (and more efficient) to define every infrastructure from the view of the application or service.

Rather than view the infrastructure as a group of resources that support a group of applications and services, we will toward a 1-1 definition of "application or service" with infrastructure.

Now reality always messes up idealistic plans, but I think this is a very important paradigm. It also leads to the notion that:

app + infrastructure + deployment = entire lifecycle

So I guess you can say we support the "application lifecycle", errr, the "transparent application lifecycle" . :)


except maybe, for shrink wrap software


So, there are my last thoughts. ..

At anyrate, I agree! Name it sooner rather than later. Besides (it can always be tweeked if it needs to be).

Alright. You get this one. I'm assigning it to you! :)
